### PR TITLE
Pagination support

### PR DESCRIPTION
**Pull request by newoldtrader**
Refactored list_product_candles_all 

Added list_accounts_all

### CODE REVIEW

File: coinbaseadvanced/client.py [hunk 1]
```diff
@@ -557,6 +594,43 @@ def get_product_candles(
...
+                begin = start_date
+            # get the next batch and extend the list
+            product_candles.candles.extend(self.get_product_candles(product_id, begin, end,granularity).candles)
+            # offset end by one granularity to avoid duplicates
+            end = begin - granularity_minutes
+            # recalculate start for the previous (older) 300 candles
+            begin = begin - (granularity_minutes_x300 + granularity_minutes)
+        
+        return product_candles
+
```
Comment (KmiQ) [inline coinbaseadvanced/client.py:633]: Looks good. Do you want to add a unit test, or do you want me to help with that before merging? We can also add the unit tests post-merging (before launching `0.2.0` release).

Comment (newoldtrader) [reply coinbaseadvanced/client.py:633]: I've looked into the existing unit tests, but I don't yet grok them. I'm not certain what is actually being tested.

I would appreciate you helping on the tests until I figure them out.

**Comment by KmiQ** [comment]
This looks good. I'm ready to get this merged (_I will follow-up with some improvements to unit tests_) as soon as you resolve the merge conflicts.

File: coinbaseadvanced/models/orders.py [hunk 2]
```diff
@@ -189,7 +189,7 @@ def __init__(self, market_market_ioc: dict = None, limit_limit_gtc: dict = None,
             **stop_limit_stop_limit_gtd) if stop_limit_stop_limit_gtd is not None else None
 
 
-class Order:
+class Order(dict):
```
Comment (KmiQ) [inline coinbaseadvanced/models/orders.py:192]: I don't think we need this but let me know if you think I'm wrong. I'm gonna merge as-is and remove it as part of my follow-up.

**Review by KmiQ — APPROVED** [review]

File: coinbaseadvanced/models/orders.py (hunk 2 above)
Comment (newoldtrader) [reply coinbaseadvanced/models/orders.py:192]: It isn't needed for this PR.

**Pull request merged**